.idea/codeStyles/Project.xml

```diff
@@ -71,7 +71,7 @@
       </class>
       <extensions>
         <pair source="cc" header="h" />
-        <pair source="c" header="h" />
+        <pair source="c" header="h"  />
```

Comment:
Thanks for reverting the changes! If possible kindly revert this whitespace change too.

Reply:
right sorry didn't notice that